stories/everLinks/src/SendSheet.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+import type BigNumber from 'bignumber.js';
+
+import { uiLocalized } from '@tonlabs/localization';
+
+import { UIBoxButton } from '@tonlabs/uikit.controls';
+import { UICardSheet } from '@tonlabs/uikit.popups';
+import { UIConstant, UIStyle } from '@tonlabs/uikit.core';
```

Comment:
We are getting rid of legacy packages. For layout tasks, use UILayoutConstant from `@tonlabs/uikit.layout`